test/cmd/get.sh

```diff
@@ -253,6 +253,48 @@ run_retrieve_multiple_tests() {
   set +o errexit
 }
 
+run_kubectl_response_compression_tests() {
+  set -o nounset
+  set -o errexit
+
+  create_and_use_new_namespace
+  kube::log::status "Testing kubectl get objects with/without response compression"
+
+  ### Test creation of large configmap objects
+  # Pre-condition: no configmaps exists
+  kube::test::get_object_assert configmaps "{{range.items}}{{${id_field:?}}}:{{end}}" ''
+  # Commands to create 3 configmaps each of size 50KB. Sum of their sizes should be >128KB (defaultGzipThresholdBytes)
+  # for apiserver to allow gzip compression of the response. This is required to test the disable-compression option
+  # from client-side
+  some_string=$(openssl rand -base64 50000)
```

Comment:
Good point and I had this same question. But looks like it's already used by some auth-related tests - https://github.com/kubernetes/kubernetes/blob/19935de5a9a4196f23f9a90cd9b7acb79091e3c2/test/cmd/authentication.sh#L57-L61

And we check there that openssl should be installed - https://github.com/kubernetes/kubernetes/blob/master/hack/lib/util.sh#L474

Reply:
something like `dd status=none if=/dev/random bs=1K count=50 | base64` might still be lighter weight

Reply:
Done, thanks for the lead.

Also I used `/dev/urandom` instead of `/dev/random` as it seems like reading the latter could be a blocking operation on some OS and output of the former suffices our purpose - https://stackoverflow.com/questions/23712581/differences-between-random-and-urandom